source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -64,9 +66,33 @@ export default class PinCode extends Component<Props> {
         this.inputsRef[inputFocusKey]
       )
         this.inputsRef[inputFocusKey].focus();
+    } else if (
+      key === 0 &&
+      this.focusKey === 0 &&
+      this.inputsRef[this.focusKey]
+    ) {
+      this.inputsRef[this.focusKey].focus();
     }
   }
 
+  onKeyPress = (evt: SyntheticKeyboardEvent<EventTarget>, key: string) => {
+    const { charCode } = evt;
+    const control: { blur?: Function, focus?: Function } = evt.target;
+    const focusKey = parseInt(key, 10);
+    const nextInputField = this.inputsRef[focusKey + 1];
+    if (charCode === 46 && nextInputField && nextInputField.focus) {
```

Comment:
I think using the human-readable key constants https://pub.dev/documentation/react/latest/react/SyntheticKeyboardEvent/key.html would make this more readable :)

Reply:
Fixed